clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/op/manifest/KubernetesDeployManifestOperation.java

```diff
@@ -124,11 +122,20 @@ public OperationResult operate(List<OperationResult> _unused) {
                     }
                   }
 
-                  setTrafficAnnotation(description.getServices(), manifest);
-                  if (description.isEnableTraffic()) {
-                    KubernetesManifestTraffic traffic =
-                        KubernetesManifestAnnotater.getTraffic(manifest);
-                    applyTraffic(traffic, manifest, inputManifests);
+                  KubernetesHandler handler =
```

Comment:
I think this `handler` instance is the same as `deployer` above, so you can reuse that.